proparse/src/main/java/org/prorefactor/core/ProToken.java

```diff
@@ -17,7 +17,9 @@
 import org.prorefactor.proparse.IntegerIndex;
 
 public class ProToken extends CommonHiddenStreamToken {
+  // TODO Why keep a reference onto this object on every ProToken ??
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1135)